src/components/new-request/search.vue

```diff
@@ -428,6 +497,26 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin, Sear
     this.scrollTo('namerequest-sbc-header')
   }
 
+  openSocietyDialog () {
+    this.showSocietiesInfo = true
+  }
+
+  closeSocietyDialog () {
+    this.showSocietiesInfo = false
+    this.setSearchBusiness(null)
+  }
+
+  get maxWidth (): string {
```

Comment:
You don't need this getter because your dialog is a fixed width.

In pick-entity-or-conversion.vue, the dialog could be really wide, ie

![image](https://github.com/bcgov/namerequest/assets/10002889/39e6e104-e500-4d89-8f5f-d0b87a0223fa)

or narrower:

![image](https://github.com/bcgov/namerequest/assets/10002889/6360564d-4428-4ae4-82e6-e0f3b766c599)

Reply:
Looks like a static value of 550px is correct for your dialog.

![image](https://github.com/bcgov/namerequest/assets/10002889/05ecf7a1-825c-4eb2-b37c-e77e6d5cd06e)

Reply:
Fixed, thanks.

Reply:
I have unresolved this conversation because you left in the `madWidth` getter that you had initially added and is no longer used. Please clean this up.

Reply:
Thanks. Resolving.